pandas/io/excel/_base.py

```diff
@@ -472,13 +479,15 @@ def read_excel(
     skipfooter: int = 0,
     storage_options: StorageOptions = None,
     dtype_backend: DtypeBackend | lib.NoDefault = lib.no_default,
+    **engine_kwargs,
```

Comment:
This is currently inconsistent with the use of engine_kwargs elsewhere in pandas (excluding SQL). Use `engine_kwargs`, not `**engine_kwargs`.